cfn-resources/api-key/mongodb-atlas-apikey.json

```diff
@@ -0,0 +1,163 @@
+{
+    "typeName": "MongoDB::Atlas::APIKey",
+    "description": "Creates one API key for the specified organization. An organization API key grants programmatic access to an organization.",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+    "definitions": {
+        "Link": {
```

Comment:
i see that many resources have this Link Options, but since is a list of objects are impossible to read, do you think we need them??

Reply:
Let it be for now, nothing harms. If we find a way to ready it might be helpful.